README.md

```diff
@@ -1 +1,649 @@
+
+| Field  | Value                                       | Description                                                    | Required |
+| ------ | ------------------------------------------- | -------------------------------------------------------------- | -------- |
+| `awv`  | `"0.1.0"`                                   | AWAKE message version                                          | Yes      |
+| `type` | `"awake/msg"`                               | Generic AWAKE message type                                     | Yes      |
+| `id`   | `sha3_256(reqStep2EcdhPk + resStep3EcdhPk)` | Message ID                                                     | Yes      |
+| `iv`   |                                             | Initialization vector for the encrypted payload                | Yes      |
+| `msg`  |                                             | Fulfilled challenge payload encrypted with AES-derived AES key | Yes      |
+
+``` javascript
+{
+  "awv": "0.1.0",
+  "type": "awake/msg",
+  "id": sha3_256(reqStep2EcdhPk + resStep3EcdhPk),
+  "iv": iv,
+  "msg": encryptedChallenge
+}
+```
+
+#### 3.4.2.2 Out-of-Band PIN Challenge
+
+Out-of-band PIN challenges are most useful when the Requestor would not be able to provide UCAN validation, such as when signing into a new device that has not been delegated to yet. The PIN MUST be set by the Responder, and transmitted out of band. Some examples of out of band transmission include displaying text on screen, email, text message, or QR code.
+
+The PIN values MUST be within the UTF-8 character set. The PIN MUST be included in the `pin` field. It is RECOMMENDED that the PIN be restricted to human-readable characters, and 4 to 10 characters long. If a very long challenge is required, it is RECOMMENDED that the SHA3 hash of the challenge be used rather than putting a large challenge over the wire.
+
+| Field  | Value                                                      | Description                 | Required |
+| ------ | ---------------------------------------------------------- | --------------------------- | -------- |
+| `did`  |                                                            | "Actual" Requestor DID      | Yes      |
+| `sig`  | `sign(requestorPK, sha3_256(responderDid + outOfBandPin))` | Signature of challenge hash | Yes      |
+
+```javascript
+{
+  "did": requestorActualDid,
+  "sig": signedHash
+}
+```
+
+#### 3.4.2.3 Direct UCAN Challenge
+
+If UCAN auth is required by the Responder, the Requestor MUST provide a UCAN. This is the same strategy as the one used by the Responder in [§3.3](#33-responder-es tablishes-point-to-point-session): the UCAN MUST be encrypted with the session key and the IV from the enclosing payload, MUST be given in a raw format, and MUST be inline (without a JSON object wrapper or similar).
```

Comment:
The link to `§3.3` is broken here

Reply:
Oh I see. The link checker probably returns a `200`, even if that specific anchor is broken. Welp, I guess back to manually checking. Thanks!

Reply:
We wouldn't have this problem with LaTeX 🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔🤔

Reply:
But only because we'd be too busy fumbling around with spacing.

![image](https://user-images.githubusercontent.com/285821/183711474-25ca6024-39e6-42aa-9452-d4a11108984c.png)